packages/utils/upgrade/src/modules/format/formats.ts

```diff
@@ -18,6 +18,10 @@ export const codemodUID = (uid: string) => {
   return chalk.bold.cyan(uid);
 };
 
+export const projectDetails = (project: AppProject | PluginProject) => {
+  return `Project: TYPE=${projectType(project.type)}, CWD=${path(project.cwd)}, PATHS=${project.paths.map(path)}`;
```

Comment:
```suggestion
  return `Project Details
Type: ${projectType(project.type)}
Cwd: ${path(project.cwd)}
Paths: ${project.paths.map(path)}
`;
```
What do you think about making this newline delimited, since PATHS is also comma delimited?

Reply:
I can maybe use a `;` instead? I was trying to avoid newlines so that one line of log = a particular context (easier to copy paste etc...)

Reply:
Ok, that makes sense, let's go with that. With all the new info we log it's tricky to keep it from feeling cluttered.

Reply:
Indeed. My goal here is to make sure running the upgrade with --debug displays enough information to fix potential issues quickly. The normal output won't be changed anyway